test/performance/Makefile.am

```diff
@@ -11,7 +11,8 @@
 
 check_PROGRAMS = $(TESTS)
 
-TESTS =
+TESTS = \
+	shmem_put_perf
```

Comment:
This should be below under ENABLE_LENGHTY_TESTS, rather than unconditionally including it in TESTS.  The test takes too long to run in the Travis environment and is causing Travis to timeout.